### ISSUE TITLE
Integration with Git?

### ISSUE THREAD

**Issue by dsmaynard** [ayushnix/pass-coffin#10]
Is there an easy workflow for using Git with pass-coffin?

**Comment by ayushnix** [comment]
Can you elaborate what you meant to say?

I'm using pass-coffin in a password store tree that is checked into a git repository and pass-coffin works fine in this case.

**Comment by dsmaynard** [comment]
Using git with standard pass, you can track changes to individual passwords, revert back to old versions and so on, and you push/pull to git simply with pass git push/pull. If pass-coffin saves the database as a tar file, how exactly do you sync this using git? Presumable pass git push/pull doesn't work anymore -- are there a set of commands specific to pass-coffin that allow you to sync with a remote git server?

If not, how exactly do you go about using pass-coffin with a password store in a git repo? Do you unlock the coffin and then use standard pass git commands?

**Comment by ayushnix** [comment]
> Do you unlock the coffin and then use standard pass git commands?

Yeah, if you want to sync changes to your password store tree using git push/pull, you'll need to unlock the repo first. The `.git/` folder is preserved in the tar file so everything works as expected after you decrypt the tar file.

Alternatively, you could backup the password store directory itself using a tool like [borg](https://www.borgbackup.org/) at regular intervals.

**Comment by dsmaynard** [comment]
Ok -- so then the the list of websites would still be visible on the remote repo, in that they are not contained within the coffin, and the coffin itself never gets pushed to the remote repo?

**Comment by ayushnix** [comment]
Yes.
